### ISSUE TITLE
About datasets

### ISSUE THREAD

**Issue by RuoyuChen10** [deeplearning-wisc/large_scale_ood#2]
Hi, thanks for your great work. I have a question about the dataset, ImageNet-1K. Do you use the full-size ImageNet-1K, which is about 160G? Or just a subset of ImageNet-1K? Thanks.

**Comment by iurgnauh** [comment]
Thanks for being interested in our work! We used the entire training set in ImageNet-1k for model training and fine-tuning, and used the entire validation set for test. The only exceptions are Fig. 2 and Fig. 5, where subsets of classes are sampled from ImageNet-1k.

Hope this answers your question.

**Comment by RuoyuChen10** [comment]
> Thanks for being interested in our work! We used the entire training set in ImageNet-1k for model training and fine-tuning, and used the entire validation set for test. The only exceptions are Fig. 2 and Fig. 5, where subsets of classes are sampled from ImageNet-1k.
> 
> Hope this answers your question.

How many GPUs did you use and how long did you train for?

**Comment by iurgnauh** [comment]
We finetuned our model from BiT pre-trained models: https://github.com/google-research/big_transfer.
We used 8 NVIDIA GeForce RTX 2080Ti and the finetuning takes about 10 hours, if I remember correctly (at least several hours, at most a day).
